src/domain/phone-provider/index.ts

```diff
@@ -5,3 +5,8 @@ export const CarrierType = {
   Void: "voip",
   Mobile: "mobile",
 } as const
+
+export const ChannelType = {
+  sms: "sms",
+  whatsapp: "whatsapp",
```

Comment:
not a blocker but in the other "enums" we use capitalized names (please check CarrierType)